store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9758,4 +9758,23 @@ TODO: delete them permanently from here
   <desc>URL of the event storage backend to be queried for event analytics</desc>
 </attr>
 
+<attr id="3046" name="zimbraAffinityEventLoggingEnabled" type="boolean" cardinality="single" optionalIn="account,cos" flags="accountInherited" callback="EventLoggerCallback" since="8.8.6">
```

Comment:
Worth including the word "contact" in the name to make it clear what the affinity is to?